docs/README.md

```diff
@@ -24,24 +24,70 @@ This repository is where the declaration of remote doctypes is done. Read more a
 
 - [Bets Unibet](com.unibet.bets): Bets from Unibet website
 
-## Metadata
+## Generic model
 
-Every doctype should have `metadata` fields.
+### Relationships
 
-- `version` is be useful for migrations
-- `dateImport` is useful for debugging purposes
+Relations between documents are under a `relationships` object at the root of the document. Relations are referenced by their names.
+
+Each relation is an object with a `data` property containing `null`, one reference or an array of references.
+
+A reference is an object containing at least a `_type` with the name of the referenced doctype and an `_id` with the id of the referenced document inside its doctype.
+
+```json
 
-```
 {
-  _id: '123456',
-  metadata: {
-    version: 1,
-    dateImport: '2018-02-22T14:54:36.861Z'
+  "_id": "mobydick",
+  "relationships": {
+    "authors": {
+      "data": [
+        { "_id": "hermanmelville", "_type": "io.cozy.contacts" }
+      ]
+    }
   }
 }
 ```
 
-## Date format
+### Document metadata
+
+We distinguish three levels : the data (a list of songs from a playlist), the metadata about the data (the creation date of the playlist itself), the metadata of the cozy document (the creation date of the cozy document describing the playlist).
```

Comment:
Using a list would make it clearer IMHO.

Reply:
three levels of what ?

Since we are in the metadata section, I would put 

We distinguish 2 levels of metadata: 

* metadata about the data (the creation date of the playlist itself)
* metadata of the cozy document (the creation date of the cozy document describing the playlist).

Reply:
1- data , 2- metadata , 3- cozy metadata